open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,33 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service
+        that they would prefer response to be paginated.
+        
+        Clients will initiate the request by sending an empty pageToken e.g. GET /tables?pageToken or /tables?pageToken=
+        For servers that support pagination, they will recognize pageToken and honor the contracts specified above 
+        by returning a NextPageToken in response if there are more results available.
+        For servers that do not support pagination, they will ignore the token and return all results.
+      required: false
+      allowEmptyValue: true
+      schema:
+        type: string
+
+    page-size:
+      name: pageSize
+      in: query
+      description:
+        For servers that support pagination, this signals an upper bound of the number of results that client will receive. 
+        For servers that do not support pagination, clients may receive results larger than the indicated pageSize.
+      required: false
+      allowEmptyValue: true
```

Comment:
what does it mean for an integer to allow empty value?

Reply:
I don't think `allowEmptyValue` applies for `pageSize`: https://swagger.io/docs/specification/describing-parameters/

We don't want to allow `GET /tables/?page-size` but rather `GET /tables/?page-size=123`, so I would just remove this

Reply:
Will remove this